### ISSUE TITLE
Websocket 403 on 1.3.4

### ISSUE THREAD

**Issue by samgurtman** [containous/traefik#1900]
BUG (1.3.4 Regression)

Upon upgrading from v1.3.3 to v1.3.4 websocket upgrades no longer correctly work. Instead of returing 101, it returns 403.

### What did you expect to see?

Websockets upgrade to return 101

### What did you see instead?

Websocket upgrade returns 403

### Output of `traefik version`: (_What version of Traefik are you using?_)

1.3.4

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
logLevel="DEBUG"

[entryPoints]
  [entryPoints.http]
  address = ":80"
    [entryPoints.http.redirect]
    entryPoint = "https"
  [entryPoints.https]
  address = ":443"
    [entryPoints.https.tls]
      [[entryPoints.https.tls.certificates]]
      certFile = "/run/secrets/site_certificate.crt"
      keyFile = "/run/secrets/site_certificate.key"

[acme]
email = "#########"
storage = "/etc/traefik/acme/acme.json"
onDemand = false
OnHostRule = true
entryPoint = "https"
dnsProvider = "cloudflare"
```

Example swarm label config:
```
      labels:
        - "traefik.frontend.entryPoints=http,https"
        - "traefik.port=80"
        - "traefik.frontend.rule=Host:#######"
        - "traefik.docker.network=australia-swarm_lb-network"
        - "traefik.backend.loadbalancer.sticky=true"
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

```
(paste your output here)
```

**Comment by Juliens** [comment]
Can you give me which websocket server you use, and which client ?

Thx

**Comment by samgurtman** [comment]
Hi there,

The server is MeteorJS. I'm not sure what it's using under the hood but something in the nodejs ecosystem. The client is  a custom ddp client.

**Comment by ldez** [comment]
@samgurtman could you provide the debug logs?

**Comment by ldez** [comment]
Closed by #1905.

**Comment by samgurtman** [comment]
Thanks for the quick follow up!
